src/ray/raylet/node_manager.cc

```diff
@@ -1013,7 +1047,7 @@ bool NodeManager::CheckDependencyManagerInvariant() const {
 }
 
 void NodeManager::TreatTaskAsFailed(const TaskSpecification &spec) {
-  RAY_LOG(DEBUG) << "Treating task " << spec.TaskId() << " as failed.";
+  RAY_LOG(INFO) << "Treating task " << spec.TaskId() << " as failed.";
```

Comment:
Is there any way to exclude the __ray_terminate__ task here? I think that is being errornously printed here.

Reply:
Well I suppose we do go through this code path for the `__ray_terminate__` task. Maybe just make this `DEBUG`?